### DISCUSSION TITLE
[Feature Request] Disable MPV User-Scripts Checkbox

### DISCUSSION THREAD

**Discussion by JordanPlayz158** [Syncplay/syncplay#512] [Archive]
I had an issue #509 and the issue was caused by my user scripts (one of them was overriding SyncPlay's CLA (Command Line Argument) for Input-IPC-Server and causing the socket to be closed causing it to fail), so I think it might be good to have an explicit checkbox for `--no-config` in SyncPlay that says "Disable user scripts (if SyncPlay is not working)" (or the like) and adds a command line argument to MPV at the beginning as `--no-config` through additional command line arguments causes it to fail which (guessing here) is due to the additional arguments being added to the end of the launch command. This would be good as --no-config isn't currently possible and can fix issues and would be easy for users to fix issues as most will assume the software is broken and stop using it if there is no easy fix.

**Comment by natto1784** [comment]
bump

**Comment by Et0h** [comment]
This discussion is being locked as there are no plans to add this feature to Syncplay. Adding non-core features not only takes up development time but can increase the complexity of the software, increasing the chance of bugs and making the interface and documentation more complex. As per #315 we only have limited development time, and so we are focussing on supporting the main use cases and not every possible way that people might want to use Syncplay.
